CHANGES.rst

```diff
@@ -1,3 +1,196 @@
+Ctapipe v0.18.0 (2023-02-09)
```

Comment:
is there any way to remove the capitalization here? Otherwise look good!

Reply:
sure